src/main/java/com/volasoftware/tinder/controller/UserController.java

```diff
@@ -30,4 +33,11 @@ public UserProfileDto getUserProfile() {
     public ResponseEntity<UserProfileDto> editUser(@Valid @RequestBody UserProfileDto userProfileDto) {
         return new ResponseEntity<>(userService.editUserProfile(userProfileDto), HttpStatus.OK);
     }
+
+    @PostMapping("api/v1/users/password-recovery")
+    public ResponseEntity forgotPassword(String email) throws MessagingException, IOException {
+        userService.getNewGeneratedPassword(email);
```

Comment:
When I see a method that starts with get... I'm expecting to receive something as return value from the method. Here the problem is more about method naming, for me